rpm/sp-rich-core.spec

```diff
@@ -0,0 +1,112 @@
+Name: sp-rich-core
+
+Version: 1.74.12
+Release: 1
+Summary: Create rich core dumps
+Group: Development/Tools
+License: GPLv2
+URL: http://github.com/mer-tools
+Source0: %{name}-%{version}.tar.gz
+BuildRequires: elfutils-libelf-devel
+BuildRequires: autoconf
+BuildRequires: gcc-c++
+Requires: sed
+Requires: coreutils
+Requires: lzop
+Requires: sp-endurance
+Requires: core-reducer
+Requires: binutils
+Requires: ssu
+Requires: ssu-sysinfo
```

Comment:
We use both. We need ssu for getting info which repositories we have enabled on device.